docs/guide/configuration/logging.md

```diff
@@ -53,9 +53,25 @@ Logging can have a significant impact on MQTT traffic. For that reason, only `in
 advanced:
   log_level: info
   log_namespaced_levels:
-    z2m:mqtt: warning
+    'z2m:mqtt': warning
 ```
 
+## Defining levels for specific namespace hierarchy
+The log levels defined in `log_namespaced_levels` will apply to the namespace itself and all namespaces below unless explicitely configured.
+In addition to the regex based debug filter [#preventing-specific-namespaces-from-being-logged] it allows to fine tune your logging needs to great detail.
```

Comment:
> The log levels defined in `log_namespaced_levels` will apply to the namespace itself and all namespaces below unless explicitely configured.
In addition to the regex based debug filter [#preventing-specific-namespaces-from-being-logged] it allows to fine tune your logging needs to great detail.

=>
Any log level defined in `log_namespaced_levels` will apply to the namespace itself and all namespaces below it, unless explicitly configured.

---

Regex part is described below and only applies to `debug`, so I don't think we need to mention it here.